### PR TITLE
Make ws client configurable

### PR DESCRIPTION
**Pull request by oneness**
This PR allows user to set policies such as idle timeout, max msg size etc.

### CODE REVIEW

File: src/clj_chrome_devtools/impl/connection.clj [hunk 1]
```diff
@@ -91,24 +91,29 @@
       (throw (ex-info "No debuggable pages found"
                       {:pages pages}))))
 
+(defn- make-ws-client [ & [{:keys [idle-timeout max-msg-size-mb]
```
Comment (tatut) [inline src/clj_chrome_devtools/impl/connection.clj]: Should `make-ws-client` be public? If the user can construct their own and give it to `connect` as parameter.

**Comment by tatut** [comment]
Thanks, I had one question comment after a quick look.

File: src/clj_chrome_devtools/impl/connection.clj (hunk 1 above)
Comment (oneness) [reply src/clj_chrome_devtools/impl/connection.clj]: Good idea. Just pushed a change to make it public along with doc string, project version and change log updates.

**Comment by tatut** [comment]
Thanks for the contribution, I'll push a new release to clojars.

**Pull request merged**